### PR TITLE
Chronos: update python3.8 support in document

### PR DESCRIPTION
**Pull request by plusbang**
## Description

During nightly test, installation options (apart from [distributed]) on Python3.8 can be passed.
We need to update the new feature in document.

### 4. How to test?
- [x] Document test
（https://binbin-doc.readthedocs.io/en/update-py-version-doc/doc/Chronos/Overview/install.html

### CODE REVIEW

File: docs/readthedocs/source/doc/Chronos/Overview/install.md [hunk 1]
```diff
@@ -19,7 +19,9 @@
 .. note::
     **Supported Python Version**:
 
-     Chronos only supports Python 3.7.2 ~ latest 3.7.x. We are validating more Python versions.
+     Chronos supports all installation options on Python 3.7.2 ~ latest 3.7.x. For details about different installation options, refer to `here <#install-using-conda>`_.
```
Comment (rnwang04) [inline docs/readthedocs/source/doc/Chronos/Overview/install.md:22]: One small question here: now chronos don't support 3.7.1？Just feel strange to this 3.7.2

File: docs/readthedocs/source/doc/Chronos/Overview/install.md [hunk 2]
```diff
@@ -19,7 +19,9 @@
...
+
+     Moreover, ``bigdl-chronos[pytorch]``, ``bigdl-chronos[tensorflow]``, ``bigdl-chronos[automl]`` and ``bigdl-chronos[inference]`` are supported on Python 3.8.
```
Comment (rnwang04) [inline docs/readthedocs/source/doc/Chronos/Overview/install.md:24]: all version of python 3.8 works？ or just a few versions like 3.8.x

**Review by rnwang04 — COMMENTED** [review]
others LGTM

File: docs/readthedocs/source/doc/Chronos/Overview/install.md (hunk 1 above)
Comment (plusbang) [reply docs/readthedocs/source/doc/Chronos/Overview/install.md:22]: > One small question here: now chronos don't support 3.7.1？Just feel strange to this 3.7.2

According to feedback from users before, there exists problems (ray support) with python3.7.0 and python3.7.1.

File: docs/readthedocs/source/doc/Chronos/Overview/install.md (hunk 2 above)
Comment (plusbang) [reply docs/readthedocs/source/doc/Chronos/Overview/install.md:24]: > all version of python 3.8 works？ or just a few versions like 3.8.x

The latest python 3.8 works.

Comment (TheaperDeng) [reply docs/readthedocs/source/doc/Chronos/Overview/install.md:24]: I think if the latest 3.8 works (and we have established a CI/CD job for it). We could claim that 3.8 is supported.

**Review by TheaperDeng — APPROVED** [review]

**Pull request merged**